src/test/java/com/coherent/training/selenium/stanila/Yandex.java

```diff
@@ -0,0 +1,90 @@
+package com.coherent.training.selenium.stanila;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import lombok.SneakyThrows;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class Yandex {
+    WebDriver driver;
+    SoftAssert softAssert;
+    public static final String USERNAME = "moraru.andy";
+    public static final String PASSWORD = "RootRoot5?";
+
+    @BeforeMethod
+    void setUp(){
+        driver =  WebDriverManager.chromedriver().create();
+        softAssert = new SoftAssert();
+    }
+
+    @Test
```

Comment:
if you want to call this function you should not call Test annotation here
you can make method which will be called in login test

Reply:
I have modified.